src/background/logging.ts

```diff
@@ -315,13 +315,13 @@ export async function _setLoggingConfig(config: LoggingConfig): Promise<void> {
 export const getLoggingConfig = liftBackground(
   "GET_LOGGING_CONFIG",
   async () => {
-    return await _getLoggingConfig();
+    return _getLoggingConfig();
   }
```

Comment:
An additional fix that this change highlights, would be to drop the wrapper function in many cases, but this might change the behavior slightly.
```suggestion
  _getLoggingConfig
```

But it can be made safe again with

```suggestion
  unary(_getLoggingConfig)
```

Want this change too, where possible?